### ISSUE TITLE
Color in legend of individual plots

### ISSUE THREAD

**Issue by biethbr1** [ggPMXdevelopment/ggPMX#57]
The legend seems to work fine for linetypes, but not colors (both are shown in the color of pop.predictions in the legend).

```
ctr <- theophylline()
ctr %>% pmx_plot_individual(
  is.legend=TRUE,
  point=list(shape=20),
  pred_line=list(linetype=6),
  pred_line=list(linetype=3))
```

(Old ticket #237)

**Comment by KZRKZR** [comment]
This is fixed in the scope of pr #160:
```
ctr <- theophylline()
ctr %>% pmx_plot_individual(
  is.legend=TRUE,
  point=list(shape=20),
  pred_line=list(color = "red",linetype=6))
```
![image](https://user-images.githubusercontent.com/10551498/107161469-9f6a5b80-69a5-11eb-9463-8e3bd3cd7cdf.png)

**Comment by KZRKZR** [comment]
@biethbr
Could we close the issue?

**Comment by tynesjo** [comment]
Fixed by #160
